index.ts

```diff
@@ -58,40 +59,40 @@ export async function isRunning(player: PlayerName) {
     }
 }
 
-/**
+/** review name
  * Kills the Spotify desktop if it's running
  */
 export function stopSpotifyIfRunning() {
     return musicCtr.stopPlayer(PlayerName.SpotifyDesktop);
 }
 
-/**
+/** review name
  * Kills the iTunes desktop if it's running
  */
 export function stopItunesIfRunning() {
     return musicCtr.stopPlayer(PlayerName.ItunesDesktop);
 }
 
-/**
+/** review name
  * Launches the Spotify desktop if it's not running
  */
 export function startSpotifyIfNotRunning() {
     return musicCtr.startPlayer(PlayerName.SpotifyDesktop);
 }
 
-/**
+/** review name
  * Launches the iTunes desktop if it's not running
  */
 export function startItunesIfNotRunning() {
     return musicCtr.startPlayer(PlayerName.ItunesDesktop);
 }
 
-/**
- * Returns the player state and track of a given player {spotify|spotify-web|itunes}
+/** review name
+ * Returns the player state and track of a given player {spotify|spotify-web|itunes }
  * - Spotify does not return a "genre"
  * - duration is in milliseconds
  * @param player
- * @returns {artist, album, genre, disc_number, duration, played_count, track_number, id, name, state}
+ * @returns {artist, album, genre, disc_number, duration, played_count, track_number, id, name, state }
  */
 export async function getState(player: PlayerName): Promise<TrackState> {
```

Comment:
"getPlayerState" might be better, especially if there is some chance that there would be a need for a "getState" function which has some other meaning.